voicemail-api/javascript/index.html

```diff
@@ -0,0 +1,231 @@
+    </script>
+    <script src="https://sdk-cdn.mypurecloud.com/javascript/83.0.0/purecloud-platform-client-v2.min.js"></script>
+    
+    <style>
+        .download-link {
+            min-width: 350px !important;
+            max-width: 350px !important;
+        }
+        .other-rows {
+            min-width: 300px !important;
+            max-width: 300px !important;
+        }
+    </style>
+</head>
+    
+<body>
+    <div class="row row-no-gutters" style="margin: 25px">
+        <div class="col-sm-5">
+            <h5><label>Choose source of voicemail you want to generate</label></h5>
+            <select name="source" id="source" class="browser-default custom-select custom-select-lg"
+                onchange="identifySelect(this)">
+                <option></option>
+                <option id="me">Me</option>
+                <option id="queue">Queue</option>
+            </select>
+        </div>
+    </div>
+
+    <div id="divQueue" class="row row-no-gutters" style="margin: 25px; display:none">
+        <div class="col-sm-5">
+            <h5><label>Select Queue</label></h5>
+            <select name="queueName" id="queueName" class="browser-default custom-select custom-select-lg"
+                onchange="getQueueVoicemails(this)">
+            </select>
+        </div>
+    </div>
+
+    <div class="row row-no-gutters" style="margin: 25px;">
+        <div id="divListContainer" style="margin: 25px;">
+            <ul class="list-group list-group-horizontal">
+                <li class="list-group-item download-link"><b>Download Link</b></li>
+                <li class="list-group-item other-rows"><b>Duration</b></li>
+                <li class="list-group-item other-rows"><b>ANI</b></li>
+                <li class="list-group-item other-rows"><b>Date and Time</b></li>
+            </ul>
+        </div>
+    </div>
+</body>
+</html>
+
+<script>
+    const clientId = process.env.GENESYS_CLOUD_CLIENT_ID;
+    const redirectUri = window.location.href;
+
+    // Set Genesys Cloud objects
+    const platformClient = require('platformClient');
+    const client = platformClient.ApiClient.instance;
+    const voicemailApi = new platformClient.VoicemailApi();
+    const routingApi = new platformClient.RoutingApi();
+
+    let voicemailDetails = "";
+
+    // Set Genesys Cloud settings
+    client.setEnvironment('mypurecloud.com');
+    client.setPersistSettings(true, 'test_app');
+
+    // Accessing Genesys Cloud objects
+    $(document).ready(() => {
+        client.loginImplicitGrant(clientId, redirectUri)
+            .then(() => {
+                console.log('Logged in');
+            })
+    });
+
+    // Identify user's selected source of voicemail
+    function identifySelect(id) {
+        let selection = id[id.selectedIndex].id;
+        return selection == "queue" ? getQueueList() :
+            getMeVoicemails();

```

Comment:
See my later comment on why we have everything as getMe instead of getMy.

Reply:
Upstream of the SDKs, nothing we can do.